features/performance/README.md

```diff
@@ -34,6 +34,18 @@ The realtime kernel will be installed using MachineConfig, which installs a new
 The template for the `MachineConfig` placed under `manifests/templates` and the script located in `assets`. The actual manifest is created by running `make generate`,
 which will base64 encode the script and put it into the template. The result is stored under `manifests/generated/11-machine-config-worker-rt-kernel.yaml`.
 
+### RT kernel configuration
+
+There are 2 env vars in the RT kernel MachineConfig template, with which you can configure:
+
+- `MICROCODE_URL`: the location of the patched microcode_ctl RPM, as long as it is not part of RHCOS yet.  
```

Comment:
can we move into the top of the file, to make it easier to see all ENV variables?

Reply:
hm, not sure, these are different: these are env vars in the MachineConfig only, nothing you can set on the fly

Reply:
I meant to add this variable to the https://github.com/openshift-kni/baremetal-deploy/tree/master/features/performance#environment-variables, just to have all ENV variable in one place, I believe you should pass these ENV variables before `make generate`

Reply:
on the other hand, maybe they should be configurable on the fly, but with defaults?

Reply:
haha, same thought at the same time

Reply:
done